internal/exec/resolvable/resolvable.go

```diff
@@ -158,6 +226,44 @@ func ApplyResolver(s *types.Schema, resolver interface{}, dirVisitors []directiv
 	}, nil
 }
 
+func buildDirectivePackers(s *types.Schema, visitors map[string]directives.Directive) (map[string]*packer.StructPacker, error) {
+	// Directive packers need to use a dedicated builder which is ready ('finish()' called) while
+	// schema fields (and their argument packers) are still being built
+	builder := packer.NewBuilder()
+
+	packers := map[string]*packer.StructPacker{}
+	for _, d := range s.Directives {
+		n := d.Name
+
+		v, ok := visitors[n]
+		if !ok {
+			// Directives which need visitors have already been checked
+			// Anything without a visitor now is an in-built directive without a packer.
```

Comment:
It might be a directive that is just "decorative". I would reword:
```suggestion
			// Directives which need visitors have already been checked.
			// Anything without a visitor now is a directive which doesn't need a packer.
```